filesystems/drs/src/test/scala/cromwell/filesystems/drs/DrsResolverSpec.scala

```diff
@@ -41,6 +41,18 @@ class DrsResolverSpec extends AnyFlatSpec with CromwellTimeoutSpec with Matchers
     DrsResolver.getContainerRelativePath(drsPath).unsafeRunSync() should be (MockDrsPaths.gcsRelativePathWithFileName)
   }
 
+  it should "find DRS path from a localization path" in {
+    val drsPath = drsPathBuilder.build(MockDrsPaths.drsPathResolvingWithLocalizationPath).get.asInstanceOf[DrsPath]
+
+    DrsResolver.getContainerRelativePath(drsPath).unsafeRunSync() should be (MockDrsPaths.gcsRelativePathWithFileNameFromLocalizationPath)
```

Comment:
TOL (Thinking Out Loud): All the other tests here are like this so I'm not asking for changes in this PR. However, I do want to point out that I think it's non-trivial to understand the relationship between the input (`MockDrsPaths.drsPathResolvingWithLocalizationPath`) and expected output (`MockDrsPaths.gcsRelativePathWithFileNameFromLocalizationPath`). Digging into `MockEngineDrsPathResolver`, I can see that they're tied together by a built-in stub response of `marthaObjWithLocalizationPath` for this input.

I would love if the input, stubbed behavior, and output were all collected together in one place. My favorite place for this is the test case itself, but maybe gathering them into a structure representing a specific scenario in `MockEngineDrsPathResolver` would work too.